### PR TITLE
Upgrade: 5.9.0

### PR DESCRIPTION
**Pull request by sumit0190**
- [x] Verified local build
- [x] Verified home page URL
- [x] Verified license identifier in SPDX
- [x] Verified package summary
- [x] Verified dev_url
- [x] Verified doc_url

Concourse build: https://concourse.build.corp.continuum.io/teams/main/pipelines/plotly_5.9.0

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: d86e44ebde38f4753dff982ab9b5e03cf872aab8fdf53a403e999ed378154331
+  sha256: b0536e72bbc0b3cf169ac1fd00759d77aae7bb12ae378cdc75c5dc362f5de576
 
 build:
   number: 0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: We should remove `noarch: python` and build it as arch-specific:
- Add skip: `True  # [py<36]`
- Use plain `python` in `run` without pinning
-  Update `test/imports` https://github.com/plotly/plotly.py/blob/ddd5ba6ba21f176914da2ee448331e05eaebb471/packages/python/plotly/setup.py#L519-L534
- Remove `six` from `run` https://github.com/plotly/plotly.py/blob/ddd5ba6ba21f176914da2ee448331e05eaebb471/packages/python/plotly/setup.py#L556 and https://github.com/plotly/plotly.py/blob/v5.9.0/CHANGELOG.md#fixed-1

See also:
1. https://github.com/plotly/plotly.py/blob/v5.9.0/packages/python/plotly/pyproject.toml (but do not add jupyterlab https://github.com/AnacondaRecipes/plotly-feedstock/pull/6#issuecomment-1059140081)

**Comment by dbast** [comment]
some dependencies are missing on s390x... win-64 still has random issues: those have to be fixed (requires more time), till then: close/-reopen maybe helps

**Comment by skupr-anaconda** [comment]
`tenacity 8.0.1` has missing artifacts for **py37-39** on `s390x`. 
https://repo.anaconda.com/pkgs/main/linux-s390x/
tenacity-8.0.1-py310ha847dfd_0.tar.bz2

That's why it failed on that platform. We need to rebuild it.

**Comment by skupr-anaconda** [comment]
I'm rebuilding tenacity 8.0.1 https://github.com/AnacondaRecipes/tenacity-feedstock/pull/3

**Comment by skupr-anaconda** [comment]
The issue with `pillow 9.2.0` on `win64` was fixed https://github.com/AnacondaRecipes/pillow-feedstock/pull/10

**Review by skupriienko — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**